docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -31,8 +31,17 @@ connector:
   - `0s` (caching disabled)
 * - `metadata.cache-missing`
   - Cache the fact that metadata, including table and column statistics, is
-    not available
+    not available.
   - `false`
+* - `metadata.schemas.cache-ttl`
+  - [The duration](prop-type-duration) for which schema metadata is cached.
```

Comment:
or do we need to say

schema names and metadata are cached

or 

schemas and schema metadata are cached

I think the first one is kinda implied in the PR wording, and the second one wrongly implies that data in the schema is cached (which is technically wrong)